### PR TITLE
Add Tizen rootfs for armel

### PR DESCRIPTION
**Pull request by hqueue**
Add a Tizen rootfs for armel and most of this commit is copied from CoreCLR.

Signed-off-by: Hyung-Kyu Choi <hk0110.choi@samsung.com>

### CODE REVIEW

**Comment by hqueue** [comment]
This is a part of core-setup issue "Enabling Linux ARM32 for .NET Core" dotnet/core-setup#725 .

File: cross/build-rootfs.sh [hunk 1]
```diff
@@ -84,6 +84,20 @@ for i in "$@" ; do
                 __UbuntuCodeName=xenial
             fi
             ;;
+        jessie)
+            __UbuntuCodeName=jessie
+            __UbuntuRepo="http://ftp.debian.org/debian/"
+            ;;
+        tizen)
+            if [ "$__BuildArch" != "arm-softfp" ]; then
```
Comment (gkhanna79) [inline cross/build-rootfs.sh]: Shouldnt this be `armel`?

Comment (hqueue) [inline cross/build-rootfs.sh]: @gkhanna79 Thanks!  I've updated it. And also change error message to "Unsupported target platform." as you suggested in CoreCLR.

**Comment by hqueue** [comment]
@gkhanna79 @mellinoe PTAL Update PR with feedback and all CI tests are passed.

**Comment by hqueue** [comment]
@gkhanna79 Yes, please merge this :)

**Pull request merged**